eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -109,6 +110,27 @@ case class NodeParams(nodeKeyManager: NodeKeyManager,
 
   /** Returns the features that should be used in our init message with the given peer. */
   def initFeaturesFor(nodeId: PublicKey): Features[InitFeature] = overrideInitFeatures.getOrElse(nodeId, features).initFeatures()
+
+  /** Returns the feerates we'd like our peer to use when funding channels. */
+  def recommendedFeerates(remoteNodeId: PublicKey, currentFeerates: FeeratesPerKw, localFeatures: Features[InitFeature], remoteFeatures: Features[InitFeature]): RecommendedFeerates = {
+    val feerateTolerance = onChainFeeConf.feerateToleranceFor(remoteNodeId)
+    val fundingFeerate = onChainFeeConf.getFundingFeerate(currentFeerates)
+    val fundingRange = RecommendedFeeratesTlv.FundingFeerateRange(
+      min = fundingFeerate * feerateTolerance.ratioLow,
+      max = fundingFeerate * feerateTolerance.ratioHigh,
+    )
```

Comment:
Where are we checking, as non-initiator, whether the funding feerate of an `open_channel` is correct?